src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -481,9 +479,10 @@ protected override void OnPropertyChanged<T>(AvaloniaPropertyChangedEventArgs<T>
             base.OnPropertyChanged(change);
         }
 
-        private void BringIntoViewRequested(object sender, RequestBringIntoViewEventArgs e)
+        private void BringIntoViewRequested(object? sender, RequestBringIntoViewEventArgs e)
         {
-            e.Handled = BringDescendantIntoView(e.TargetObject, e.TargetRect);
+            if (e.TargetObject is not null)
```

Comment:
A bit weird that TargetObject can be null when we want to bring it into the view

Reply:
Yeah this is mainly due to the fact that a `RequestBringIntoViewEventArgs` event can be sent by anyone, and they're not guaranteed to have set the `TargetObject` property. In that case it's either throw or ignore, and throwing deep in a bubbling/tunneling event handler isn't a great idea in general.